automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Modal.scala

```diff
@@ -33,6 +33,7 @@ class OKCancelModal(id: String)(implicit webDriver: WebDriver) extends Modal(id)
   protected val cancelButton = Button(innerElement("cancel-button"))
 
   override def awaitReady(): Unit = {
+    header.awaitVisible()
```

Comment:
use Modal header to identify correct modal is open. otherwise running into risks of working in a wrong Modal

Reply:
I don't think that's true. The `okButton` is bound to be inside of this modal, so it can't match another modal.

Reply:
Ah, right. When reading this, I asked myself how we know the header is specific to this modal. The answer is `innerElement`. Since `innerElement` is also used for `okButton`, we should be just as certain that about it being the right button as we are it being the right header, making the header check unnecessary.

Reply:
As a measure of precaution! If Header check failed, stacktrace shows what Modal test is expecting by logging Header cssSelector. `okButton` timeout doesn't show what Modal header is. It will be super useful for debugging Modal failure.